libraries/standard/mqtt/src/mqtt.c

```diff
@@ -1468,9 +1468,12 @@ uint16_t MQTT_GetPacketId( MQTTContext_t * pContext )
     if( pContext != NULL )
     {
         packetId = pContext->nextPacketId;
-        pContext->nextPacketId++;
 
-        if( pContext->nextPacketId == 0U )
+        if( pContext->nextPacketId == UINT16_MAX )
```

Comment:
I think we missed it earlier, it would be nice to add a reason here the the packet-Id 0 is  invalid.

Reply:
Noted, I will add with the next wave of MQTT changes.